index.html

```diff
@@ -1,14 +1,284 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<!-- Basic Page Setup -->
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href='/mockUpCss/index.css' rel='stylesheet'>
+</head>
+
+<!-- Full Layout -->
+
+<body>
+
+  <!-- Top Section -->
+  <section id="header" class="centered-titles">
+    <h1><a href="/index.html">
+        HOME PAGE
+      </a></h1>
+    <h6>
+      VILLAINS ANONYMOUS
+    </h6>
+  </section>
+
+  <!-- Top Features -->
+  <h5 class="centered-titles">
+    WINNERS OF VILLAINS ANONYMOUS AWARDS
+  </h5>
+
+  <!-- Feature Pictures -->
+  <section id="image-box">
+    <img src='/images/sauron.webp' class="flex-pictures" id="flex-pic-1">
+    <img src='/images/vader.jpg' class="flex-pictures">
+    <img src='/images/smith.jpg' class="flex-pictures" id="flex-pic-3">
+  </section>
+
+  <!-- Feature Title -->
+  <section id="image-titles">
+    <div>
+      MOST POWERFUL
+    </div>
+    <div>
+      OVERLORD OF THE CENTURY
+    </div>
+    <div>
+      BEST IN TECH
+    </div>
+  </section>

```

Comment:
Combining your caption and image into the same div would make it easier to center your caption under the image. As you have it right now the middle and left ones are slightly off centered. This would contain each one to be centered on the image